test/ElasticSearchTest.php

```diff
@@ -257,6 +257,8 @@ public function testCreateIndexes_callIndexCreationBasedOnIndexOption(): void
             ->method('indices')
             ->willReturn($indexMock);
 
+        $this->sut->setOption('indexFiles', './vendor/oat-sa/lib-tao-elasticsearch/test/sample/testIndexes.conf.php');
+
         $this->sut->createIndexes();
```

Comment:
`sut`? Not very explicit name

Reply:
This is a test standard name "Subject of test". In the unit test is always the class being tested

Reply:
I did not reformat all code in order to keep this PR smaller